megazords/ios-rust/Cargo.toml

```diff
@@ -6,7 +6,7 @@ authors = ["Sync Team <sync-team@mozilla.com>"]
 license = "MPL-2.0"
 
 [lib]
-crate-type = ["staticlib"]
+crate-type = ["staticlib", "rlib"]
```

Comment:
Had to add `rlib` so I can reference it inside `swift-components-docs`. This will increase the compile times.